docs/installation.rst

```diff
@@ -67,6 +67,29 @@ support this::
 
     pip install scales
 
+*Optional:* Column-Level Encryption (CLE) Support
+--------------------------------------------------
+The driver has built-in support for client-side encryption and
+decryption of data. Please see :doc:`column_encryption` for more detail
+about this feature, including how to configure and use it.  This
+feature relies on the `cryptography <https://cryptography.io/en/latest/>`_ module
+for its implementation. The initial release of the CLE feature in Python driver 3.27.0
+explicitly listed cryptography as a required dependency; as a result the module will be
+automatically downloaded and installed when that version of the driver is installed.
+Later versions of the driver do not specify this module as a requirement, so if you
+want to use the CLE feature with a version of the driver after 3.27.0 you will need
+to take additional steps to install the cryptography module.
```

Comment:
I could see this change being a source of confusion so I wanted to very explicitly spell out the differences between the versions.  It's a bit more verbose but I'd argue this is a case where we want to provide a bit more information to make the constraints clear to our users.